tests/NHSD.BuyingCatalogue.API.IntegrationTests/Features/Road Map/Supplier Get Solution Road Map.feature

```diff
@@ -22,7 +22,7 @@ Scenario: Solution RoadMap is retrieved for the solution
     And the string value of element summary is some valid summary 
  
 @3664
-Scenario: Solution RoadMap is retrieved empty for the solution where no solution detail exists
+Scenario: Solution RoadMap is retrieved empty for the solution where no solution RoadMap exists
```

Comment:
Suggestion: `solution road map summary is not retrieved when no road map exists`.

Reply:
Updated scenario details to `solution road map summary is not retrieved when no road map exists`